js/services/validation.ts

```diff
@@ -3,7 +3,7 @@ import * as Model from './model';
 import * as Settings from './settings';
 import * as Utils from './utils';
 
-const isValueDefined = function (value: any, contentType: string) {
+const isValueNotDefined = function (value: any, contentType: string) {
```

Comment:
This doesn't seem right to me. Having the function check for a "not" positive is confusing.

Reply:
Sorted